shell/gpu/BUILD.gn

```diff
@@ -88,3 +88,24 @@ source_set("gpu_surface_metal") {
     public_deps += [ "//flutter/impeller" ]
   }
 }
+
+executable("gpu_surface_unittests") {
+  testonly = true
+  sources = []
+
+  if (is_mac) {
+    sources += [ "gpu_surface_metal_impeller_unittests.mm" ]
+  }
+
+  if (is_mac) {
```

Comment:
Consider combining back-to-back `if (is_mac)`

Reply:
Done